examples/raytracing_sandbox.html

```diff
@@ -283,25 +283,12 @@
 
 				}
 
-				info.innerHTML = '<a href="http://threejs.org" target="_blank" rel="noopener">three.js<a/> - raytracing renderer (using '  + WORKERS + ' <button onclick="updateWorkers(-1)">-</button><button onclick="updateWorkers(1)">+</button> web workers)' +
+				info.innerHTML = '<a href="http://threejs.org" target="_blank" rel="noopener">three.js<a/> - raytracing renderer (using ' + WORKERS + ' <button onclick="updateWorkers(-1)">-</button><button onclick="updateWorkers(1)">+</button> web workers)' +
 				'<br/><button onclick="rearrange()">Rearrange</button><button onclick="render()">Render</button>';
 
 
 			}
 
-			function rearrange() {
```

Comment:
This function is called in html on [line 287](https://github.com/mrdoob/three.js/blob/4ebbdc8f64fc7f2e9f8bd2c928136ce9b0f63e94/examples/raytracing_sandbox.html#L287)

Reply:
Oh, good catch!